app/services/duplicate_application.rb

```diff
@@ -26,6 +26,18 @@ def duplicate
         )
       end
 
+      if FeatureFlag.active?(:one_personal_statement) && !original_application_form.single_personal_statement?
+        original_becoming_a_teacher = original_application_form.becoming_a_teacher
+        original_subject_knowledge = original_application_form.subject_knowledge
+        merged_personal_statement = "#{original_becoming_a_teacher}#{$INPUT_RECORD_SEPARATOR}#{$INPUT_RECORD_SEPARATOR}#{original_subject_knowledge}"
```

Comment:
Any reason not to use `\n` for the newlines?

```suggestion
        merged_personal_statement = "#{original_becoming_a_teacher}\n\n#{original_subject_knowledge}"
```